js/Data/MigrationPatterns.ts

```diff
@@ -55,7 +55,6 @@ export enum MigrationNodeId {
     'West Virginia',
     Wisconsin,
     Wyoming,
-    'Puerto Rico',
```

Comment:
Why was this removed?  I know Puerto Rico wasn't in the map, but there's a way to get it there

Reply:
There was no economic data for Puerto Rico or DC, and since they don't really show up on the map anyway I figured we could just take them off. I thought it might be odd in the table too if they don't have any data other than immigration data